src/main/java/edu/kit/provideq/toolbox/maxcut/MaxCutController.java

```diff
@@ -19,7 +20,7 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-public class MaxCutController extends ProblemController<String, String, MaxCutSolver> {
+public class MaxCutController extends ProblemController<String, Gml, MaxCutSolver> {
```

Comment:
Why do we return an object? our gml-object format (json-ed, not toString-ed) is non-standard

Reply:
As we discussed this wednesday, we'll focus on returning standards-compliant formats first. The parsing comes with only a minimal overhead since the actual work lies in the solution of the contained problem